webpack.sample.config.js

```diff
@@ -33,7 +33,7 @@ require.extensions['.webapp'] = function (module, filename) {
 };
 const manifest = require('./app/manifest.webapp');
 
-const THIS_APP_ID = 'openmrs-addonmanager';
+const THIS_APP_ID = 'Add On Manager';
```

Comment:
Why not make the id as simple as? addonmanager

Reply:
Done